packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -666,15 +662,21 @@ export async function createHotReloaderTurbopack(
           }
         }
 
-        const sync: SyncAction = {
-          action: HMR_ACTIONS_SENT_TO_BROWSER.SYNC,
-          errors,
-          warnings: [],
-          hash: '',
-          versionInfo,
-        }
+        ;(async function () {
+          const versionInfo = await getVersionInfo(
```

Comment:
is this delying every time now for sync event?
Maybe we should call the get version at the very beginning? and only send the actual version when the promise is resolved.

Reply:
Great call, thank you!